automation/project/Dependencies.scala

```diff
@@ -7,7 +7,7 @@ object Dependencies {
   val akkaV = "2.5.7"
   val akkaHttpV = "10.0.10"
 
-  val serviceTestV = "0.16-e73039a"
+  val serviceTestV = "0.16-a753e32-SNAP"
```

Comment:
It is the local published snapshot. test won't work in Jenkin until merge [workbench-lib #220](https://github.com/broadinstitute/workbench-libs/pull/220)